ref/Castle.Core-net45.cs

```diff
@@ -327,7 +327,7 @@ public interface IBindingList<T> : Castle.Components.DictionaryAdapter.IBindingL
         bool SupportsChangeNotification { get; }
         bool SupportsSearching { get; }
         bool SupportsSorting { get; }
-        public event System.ComponentModel.ListChangedEventHandler ListChanged;
+        event System.ComponentModel.ListChangedEventHandler ListChanged;
```

Comment:
Unexpected?

Reply:
I'd say "correctness": this is a declaration for an interface member, so `public` is implied and shouldn't have been there to begin with.

Reply:
Yes, I stupidly misread it as a member of `GroupAttribute` in the diff.